x-pack/plugins/security_solution/public/timelines/store/timeline/helpers.ts

```diff
@@ -153,7 +154,7 @@ interface AddNewTimelineParams {
 export const addNewTimeline = ({
   columns,
   dataProviders = [],
-  dateRange = { start: 0, end: 0 },
+  dateRange: mayDateRange,
```

Comment:
nit: may or my? ;)

Reply:
Its `may` because it can be `undefined`. `maybeDateRange` better?

Reply:
yes because May is a month, thats how i read it lol. not worth a new build if you get green

Reply:
lol, ok! I am gonna fix it for sure in another PR.